src/components/cards/CardItem.vue

```diff
@@ -122,6 +123,10 @@ export default {
 		currentCard() {
 			return this.card && this.$route && this.$route.params.cardId === this.card.id
 		},
+		
+    () {
```

Comment:
```suggestion
		labelsSorted() {
``` ?

Reply:
I don't know what the problem here is. In my IDE it's correct:
![grafik](https://user-images.githubusercontent.com/24530680/93856559-73ca2b80-fcb9-11ea-9340-c5b0f8dc7d45.png)